node/config.go

```diff
@@ -33,7 +33,7 @@ var (
 		0: "eth_quorum",
 		1: "permissioned_quorum",
 	}
-	SemVer    = "v0.0.0"
```

Comment:
How is this used today and what difference does this make?

Reply:
This is a fallback. Only gets triggered if you do a `go build` without passing `-ldflags="-X 'github.com/Layr-Labs/eigenda/node.SemVer=${SEMVER}'` so pretty much just our unit tests will see `0.0.0`

I'm updating it to be semver spec compliant which does not use `v` prefix.